backend/session/logind.c

```diff
@@ -672,6 +816,27 @@ static struct wlr_session *logind_session_create(struct wl_display *disp) {
 		goto error_bus;
 	}
 
+	// Check for CanGraphical first
+	sd_bus_error error = SD_BUS_ERROR_NULL;
+	ret = sd_bus_get_property_trivial(session->bus,
+		"org.freedesktop.login1", session->seat_path,
+		"org.freedesktop.login1.Seat", "CanGraphical", &error,
+		'b', &session->can_graphical);
+	if (ret < 0) {
+		wlr_log(WLR_ERROR, "Failed to get 'CanGraphical' property: %s",
+			error.message);
+		sd_bus_error_free(&error);
+		goto error_bus;
+	}
```

Comment:
Wouldn't it be better to use `sd_seat_can_graphical` instead of calling out to dbus here?

Reply:
Yes that sounds better. Changed.